docs/README.md

```diff
@@ -1,21 +1,20 @@
-<!--
-Licensed to the Apache Software Foundation (ASF) under one
-or more contributor license agreements.  See the NOTICE file
-distributed with this work for additional information
-regarding copyright ownership.  The ASF licenses this file
-to you under the Apache License, Version 2.0 (the
-"License"); you may not use this file except in compliance
-with the License.  You may obtain a copy of the License at
+[//]: # Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
gatsby-mdx was tripping on the comment...

Reply:
Hah, I have [another PR](https://github.com/apache/incubator-superset/pull/10898) open for some redirect support, and it seems like you can get away with just the `# ` for comments. Well, we'll see if it passes CI.

Reply:
Hmmm. The comment format in this PR is still visible on the readme too... not sure what the trick is here.

![image](https://user-images.githubusercontent.com/812905/93306257-7659ec00-f7b4-11ea-8c51-c924679429fb.png)